src/Orleans.Runtime/Placement/ResourceOptimizedPlacementDirector.cs

```diff
@@ -204,13 +204,13 @@ private bool IsLocalSiloPreferable(IPlacementContext context, SiloAddress[] comp
         float normalizedCpuUsage = stats.CpuUsage / 100f;
         float score = _weights.CpuUsageWeight * normalizedCpuUsage;
 
-        if (stats.TotalPhysicalMemory > 0)
+        if (stats.MaxAvailableMemory > 0)
         {
-            long physicalMemory = stats.TotalPhysicalMemory; // cache locally
+            long maxAvailableMemory = stats.MaxAvailableMemory; // cache locally
 
-            float normalizedMemoryUsage = stats.MemoryUsage / physicalMemory;
-            float normalizedAvailableMemory = 1 - stats.AvailableMemory / physicalMemory;
-            float normalizedPhysicalMemory = PhysicalMemoryScalingFactor * physicalMemory;
+            float normalizedMemoryUsage = stats.MemoryUsage / maxAvailableMemory;
+            float normalizedAvailableMemory = Math.Max(0, 1 - stats.AvailableMemory / maxAvailableMemory);
```

Comment:
@ReubenBond is there a need for `Max()`? 
`1 - stats.AvailableMemory / maxAvailableMemory` can not be negative AFAIK